migration/v0.37/README.md

```diff
@@ -41,14 +41,14 @@ As with all scripts in this repository `CK8S_CONFIG_PATH` is expected to be set.
 - [ ] Check if there are any pending changes to the environment;
 - [ ] Check the state of the environment, pods, nodes and backup jobs:
 
-    ```bash
-    ./bin/ck8s test sc|wc
-    ./bin/ck8s ops kubectl sc|wc get pods -A -o custom-columns=NAMESPACE:metadata.namespace,POD:metadata.name,READY-false:status.containerStatuses[*].ready,REASON:status.containerStatuses[*].state.terminated.reason | grep false | grep -v Completed
-    ./bin/ck8s ops kubectl sc|wc get nodes
-    ./bin/ck8s ops kubectl sc|wc get jobs -A
-    ./bin/ck8s ops helm sc|wc list -A --all
-    velero get backup
-    ```
+  ```bash
```

Comment:
It looks like all these changes comes from the old template. Maybe you can rebase and only add the things you've added about the kube-prometheus-stack migration to solve this
```suggestion
    ```bash
```

Alternatively you have a reason for these changes